src/BCReader.jl

```diff
@@ -196,8 +195,9 @@ function update_midmonth_data!(date, bcf_info::BCFileInfo{FT}) where {FT}
         bcf_info.monthly_fields[2] .= deepcopy(bcf_info.monthly_fields[1])
         bcf_info.segment_length .= Int(0)
 
-    elseif Dates.days(date - all_dates[end - 1]) > 0 # for fini
-        @warn "this time period is after BC data - using file from $(all_dates[end - 1])"
+        # case 2: date is after the last date in file
+    elseif (date - all_dates[end]).value > 0 # for fini
```

Comment:
I think we can remove the comment "for fini" - not sure what that's meant to say